ptf_tests/common/utils/test_utils.py

```diff
@@ -37,144 +37,196 @@
 
 
 def add_port_to_dataplane(port_list):
-    """
```

Comment:
please check this file. seems all required indentations are removed

Reply:
updated